compute_tools/src/compute.rs

```diff
@@ -532,6 +532,50 @@ impl ComputeNode {
         Ok(())
     }
 
+    /// Start and stop a postgres process to warm up the VM for startup.
+    pub fn prewarm_postgres(&self) -> Result<()> {
+        info!("prewarming");
+
+        // Create pgdata
+        let pgdata = &format!("{}.warmup", self.pgdata);
+        create_pgdata(pgdata)?;
+
+        // Run initdb to completion
+        info!("running initdb");
+        let initdb_bin = Path::new(&self.pgbin).parent().unwrap().join("initdb");
+        Command::new(initdb_bin)
+            .args(["-D", pgdata])
```

Comment:
There is also an option `--no-sync` to don't fsync data on disk. Could be a good idea to add it to make it faster

Reply:
Ok. Right now aiming for maximizing signal (is this a good idea or not), so trying to avoid any differences from the real postgres process that runs later

Reply:
> so trying to avoid any differences from the real postgres process that runs later

Real process doesn't run `initdb`, but up to you